dev/sparktestsupport/modules.py

```diff
@@ -241,7 +250,7 @@ def contains_file(self, filename):
 
 pyspark_core = Module(
     name="pyspark-core",
-    dependencies=[mllib, streaming, streaming_kafka],
+    dependencies=[mllib, streaming, streaming_kafka, streaming_flume],
```

Comment:
@JoshRosen I'm not sure if this change is right. Could you explain why `pyspark_core` depends on `mllib, streaming, streaming_kafka`?